packer/linux/buildkite-ami.json

```diff
@@ -58,29 +54,11 @@
     },
     {
       "type": "shell",
-      "script": "scripts/install-s3secrets-helper.sh"
-    },
-    {
-      "type": "shell",
-      "script": "scripts/install-git-lfs.sh"
-    },
-    {
-      "type": "shell",
-      "script": "scripts/install-session-manager-plugin.sh"
-    },
-    {
-      "type": "shell",
-      "script": "scripts/install-nvme-cli.sh"
```

Comment:
Moved to https://github.com/buildkite/elastic-ci-stack-for-aws/blob/8752ac0ff6e4125b8231b83fc683939ef58bfc17/packer/linux/scripts/install-utils.sh#L20